backend/algorithms/cache/cache_config.json

```diff
@@ -0,0 +1,50 @@
+{
+    "codes": [
+        "ACTL#",
+        "ASCI#",
+        "BUSN#",
+        "COMM#",
+        "COMP#",
+        "DATA#",
+        "ECON#",
+        "INFS#",
+        "MATH#",
+        "ZBUS#",
+        "SOSS#",
+        "MDIA#",
+        "CRIM#",
+        "LAWS#",
+        "INST#",
+        "SOSS#",
+        "DDES#",
+        "ARTS#",
+        "DART#",
+        "MUSC#",
+        "SOCW#",
+        "PPEC#",
+        "EDST#" 
+    ],
+    "keyword_mapping": {
```

Comment:
Since these keys are being `.lowered()` anyway, I think we may as well capitalize the keys to reflect how they appear in the handbook

Reply:
Do you mean just move them all to `Title Case`?

Reply:
Yeah pretty much

Reply:
changes made